engine/src/main/java/org/terasology/rendering/dag/RenderGraph.java

```diff
@@ -50,7 +50,14 @@ public void addNode(Node node) {
         }
 
         if (graph.nodes().contains(node)) {
```

Comment:
Ok, I _think_ the flow will never get here. Because if I attempt to add a node twice to the graph, it will get caught by the nodeUri check. Do you confirm?

Reply:
Ok, this has been addressed.